lib_methods.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+# rubocop:disable Metrics/ModuleLength
+module Actions
+  def list_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'No person available'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)?: '
+    user_input = gets.chomp.to_i
```

Comment:
- [ ] Hi @Okechukwu-muokwugwo, I think this is not a good example of the `Single responsibility rule` which state that a method/class should have only one job. All requests for inputs should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request input from the user.